ppr-ui/src/composables/mhrRegistration/home-owner.ts

```diff
@@ -94,11 +94,18 @@ export function useHomeOwners (isPerson: boolean = false, isEditMode: boolean =
     const fractionalDenominator = getMhrRegistrationHomeOwnerGroups.value[0]?.interestTotal || null
 
     return {
-      totalAllocation: totalFractionalNominator + ' / ' + fractionalDenominator,
-      hasTotalAllocationError: totalFractionalNominator !== fractionalDenominator
+      totalAllocation: totalFractionalNominator + '/' + fractionalDenominator,
+      hasTotalAllocationError: totalFractionalNominator !== fractionalDenominator,
+      hasMinimumGroupsError: getMhrRegistrationHomeOwnerGroups.value.length < 2
     }
   }
 
+  const getInterestString = (): string => {
+    return getMhrRegistrationHomeOwnerGroups !== undefined && getMhrRegistrationHomeOwnerGroups.value.length >= 1 ? getMhrRegistrationHomeOwnerGroups.value[0].interest : ''
+  }
+  const getNumberOfGroups = (): boolean => {
+    return !getMhrRegistrationHomeOwnerGroups.value || getMhrRegistrationHomeOwnerGroups.value.length < 2
+  }
```

Comment:
The property name is a little misleading, `getNumberOfGroups` to return a `boolean`. Maybe we could better describe what this expression is doing in the name. ie `hasMinimumGroups` or whatever

Reply:
good catch